v3/src/components/case-table/case-table-registration.ts

```diff
@@ -20,6 +22,10 @@ registerTileContentInfo({
   prefix: kCaseTableIdPrefix,
   modelClass: CaseTableModel,
   defaultContent: () => ({ type: kCaseTableTileType }),
+  getTitle: (tile) => {
+    const data = tile?.content && getTileDataSet(tile?.content)
+    return data?.title || t("DG.DocumentController.caseTableTitle")
```

Comment:
Yes. For a case table/card the title is always based on the title of the data context. Therefore, when we change a table/card title, we change the title of the data context instead of just the tile.